### ISSUE TITLE
Jupyter Lab V2.1.0 - Extension Manager -  Blank Page (Similar issues #7171 &  #8122)

### ISSUE THREAD

**Issue by saboomayank** [jupyterlab/jupyterlab#8231]
## Description

Jupyter Lab  Extension Manager shows blank page even though an extension is installed. Cannot search extensions as search bar has disappeared.

![Screenshot from 2020-04-19 22-36-17](https://user-images.githubusercontent.com/10496596/79717914-be212980-828f-11ea-8f0d-7a03225c8189.png)

Similar to issues  #7171 and #8122 

However, following their advised solutions to delete the `build_config.json` did not help.

Also, do check the console output.

Thanks!

## Context

- Operating System and version: Ubuntu 19.10
- Browser and version: Mozilla 75.0
- JupyterLab version: 2.1.0

<details><summary>Browser Output</summary>
<pre>
starting application in workspace: "/lab" index.js:50
SyntaxError: "JSON.parse: unexpected character at line 1 column 1 of the JSON data" listings.js:46
    Lister listings.js:46
Use of Mutation Events is deprecated. Use MutationObserver instead. index.js:29
Starting WebSocket: ws://localhost:8888/api/kernels/bb39d318-67bd-4090-963b-b2c2795cb5f3 2 default.js:50

![Screenshot from 2020-04-19 22-54-28](https://user-images.githubusercontent.com/10496596/79718375-c5950280-8290-11ea-9602-2114b6fe2ba2.png)

</pre>
</details>

**Comment by jtpio** [comment]
Thanks @saboomayank.

This seems to be reproducible on [Binder](https://mybinder.org/v2/gist/jtpio/a178939b687ec421e76113bc9b18d33a/master?urlpath=%2Flab) too using the following `environment.yml`:

```yaml
name: jupyterlab-2
channels:
- conda-forge
dependencies:
- jupyterlab=2
- nodejs
```

From this gist: https://gist.github.com/jtpio/a178939b687ec421e76113bc9b18d33a

This pulls JupyterLab 2.1.

However it seems to be fine with JupyterLab 2.0.x:

![image](https://user-images.githubusercontent.com/591645/79745530-c9fd0200-8308-11ea-92f3-4b98f084c7cd.png)

It might be related to the recent rework of the extension manager?

**Comment by saboomayank** [comment]
Thank you @jtpio 

It looks like it might be a rework of the extension manager causing the error. I plan on downgrading to version 2.0 and see if that solves the issue.

**Comment by saboomayank** [comment]
**UPDATE @jtpio 
Jupyter Lab v2.0.2 works fine. I haven't been able to isolate the problem in v2.1.

![Screenshot from 2020-04-20 16-49-24](https://user-images.githubusercontent.com/10496596/79810065-fcb5f300-8326-11ea-806d-2ce6dc013978.png)

Thank you for your help!

Stay safe.

PS- I am going to go ahead and close the issue.

**Comment by jtpio** [comment]
Thanks @saboomayank for checking.

This still sounds like an issue with 2.1 though.

**Comment by jtpio** [comment]
This looks it is caused by the API call to `/lab/api/listings/@jupyterlab/extensionmanager-extension/listings.json`, which doesn't return valid JSON:

![image](https://user-images.githubusercontent.com/591645/79861934-845a3b00-83d5-11ea-80a0-07f689f518a1.png)

**Comment by saboomayank** [comment]
@jtpio Thank you for noticing that.

**Comment by enriqueartal** [comment]
Still happens in 2.1.1

**Comment by jtpio** [comment]
@enriqueartal which version of `jupyterlab_server` is installed? Does upgrading to `1.1.0` help fix the issue?

**Comment by enriqueartal** [comment]
It was 1.1.0, and I have just upgraded to 1.1.1 and nothing changed.

**Comment by vidartf** [comment]
@enriqueartal Do you get the same error in the browser console as was reported in the OP?

**Comment by enriqueartal** [comment]
I am not sure. I attach the log.
[localhost-1588327312078.log](https://github.com/jupyterlab/jupyterlab/files/4563402/localhost-1588327312078.log)

**Comment by enriqueartal** [comment]
Using kde instead of gnome and it works!

**Comment by qiemem** [comment]
I ran into this on 2.2.6 and fixed it by downgrading to 2.0.2.
